frontend/src/components/pages/club_display/ClubDescription.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    boldTextElement: {
+      fontWeight: 'bold',
+      marginBottom: theme.spacing(0),
+    },
+    textElement: {
+      marginBottom: theme.spacing(0),
+      marginTop: theme.spacing(0),
+    },
+  })
+);
+
+interface ClubDescriptionProps {
+  description: string,
+};
```

Comment:
```suggestion
}
```
There shouldn't be a semicolon here.